packages/botonic-react/tests/components/multichannel/multichannel-button.test.jsx

```diff
@@ -44,4 +47,24 @@ describe('Multichannel buttons:', () => {
     const rendered = whatsappRenderer(sut).toJSON()
     expect(rendered).toMatchSnapshot()
   })
+
+  test('with webview', () => {
+    const sut = (
+      <MultichannelButton {...Buttons.withWebview.props}>
```

Comment:
following the same approach as the other ones, and as we talked, maybe it would be better to leave it as:
```
<MultichannelButton webview='webview'>
button text with Webview
</MultichannelButton>
```

Reply:
Totally agree, refactored!